docs/internals/service-management.md

```diff
@@ -42,6 +48,17 @@ In the above service graph we start the `DatabaseService` first, the `SchemaMigr
 second, and finally the `MovieService`. The `MovieService` doesn't need to express a dependency
 on the `SchemaMigrationService`, that happens automatically for enhancements.
 
+## The ReadyService
+This is a symbolic service that's useful to define the relationship, generally, between services which process traffic (i.e. Jetty) 
+and infrastructure services (i.e. persistence stores).
+
+By having the former depend on ReadyService and the latter enhanced by ReadyService we can force, for example, 
+JettyService to stop before our feature flag service without having to intertwine our dependency graph.
+
+In general, infrastructure services should enhance ReadyService and application services should depend on ReadyService.
+
+See more information here [here](..%2F0.x%2Fmisk-service%2Fmisk-service%2Fmisk%2F-ready-service%2Findex.md).
```

Comment:
Is this URL link right? it looks urlencoded and it probably shouldn't be

Reply:
right, will fix it, thanks!